clients/extension/src/pages/popup/pages/import-file/index.tsx

```
@@ -0,0 +1,304 @@
+      error: message,
+    }))
+  }
+
+  const onFileSelected = (file: File) => {
+    setState(prevState => ({ ...prevState, file }))
+    mutate(shouldBePresent(file))
+  }
+
+  useEffect(() => {
+    finalizeVaultImport()
+  }, [vault, vaultContainer])
```

ESLint

[error] React Hook useEffect has a missing dependency: 'finalizeVaultImport'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)